cartoframes/client/data_obs_client.py

```diff
@@ -0,0 +1,692 @@
+from __future__ import absolute_import
+
+import json
+import requests
+import collections
+import pandas as pd
+from warnings import warn
+
+from carto.exceptions import CartoException
+
+from ..data.dataset import Dataset
+from ..data.utils import get_countrytag
+
+from .. import utils
+from ..context import create_context
+
+
+class DataObsClient(object):
+    """
+    Data Observatory v1 class. `Data Observatory documentation
+    <https://carto.com/developers/data-observatory/>`__.
+
+    This class provides the following methods to interact with Data Observatory:
+      - boundary: returns a :py:class:`Dataset <cartoframes.data.Dataset>` with the world boundaries found.
```

Comment:
Instead of `world boundaries found.` it should be `geographic boundaries (geometries) or their metadata`